airbyte-integrations/connectors/source-surveymonkey/source_surveymonkey/streams.py

```diff
@@ -109,52 +134,33 @@ def request_params(self, stream_state: Mapping[str, Any], **kwargs) -> MutableMa
         params["start_modified_at"] = since_value.strftime("%Y-%m-%dT%H:%M:%S")
         return params
 
-    def get_updated_state(self, current_stream_state: MutableMapping[str, Any], latest_record: Mapping[str, Any]) -> Mapping[str, Any]:
-        """
-        Return the latest state by comparing the cursor value in the latest record with the stream's most recent state object
-        and returning an updated state object.
-        """
-        return {self.cursor_field: max(latest_record.get(self.cursor_field), current_stream_state.get(self.cursor_field, ""))}
 
+class SliceMixin:
```

Comment:
nice mixin usage 👍 
Maybe rename to `SurveyIDSliceMixin`, or similar, for clarity

Reply:
updated